scipy/signal/windows/windows.py

```diff
@@ -1620,6 +1620,80 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(M, nbar=4, level=-30, norm=True, sym=True):
+    """
+    Return the Taylor window.
+
+    The Taylor window allows for a selectable sidelobe suppression with a
+    minimum broadening. This window is commonly used in radar processing [1]_.
```

Comment:
Should mention it's an approximation of chebwin

Reply:
This line has `[1]_` which produces a link. There is no `[2]_` so no link.